### ISSUE TITLE
Can not delete resource from the list.

### ISSUE THREAD

**Issue by atompie** [Tracardi/tracardi-gui#203]
Steps to reproduce
--------------------------------

1. [Install the system](http://docs.tracardi.com/development/react_env/)
2. Login an go to Resources and click icon 
3. You should see this
![Screenshot from 2022-07-28 12-30-07](https://user-images.githubusercontent.com/16271564/181484739-84433a0c-f084-4d8b-83e1-14c8eae0ea19.png)

The list does not have the trash icon on the list. So it can not be deleted from here as in for example Maintenance/Action Plugins.
![Screenshot from 2022-10-04 11-59-39](https://user-images.githubusercontent.com/16271564/193791763-6ee50555-2aa1-4fe2-9984-2f5c28de47d8.png)
It can be deleted in details, though

Everything on the backed side is ready we need to add the icon to the list. See `src/components/pages/Resources.js` and the part:

```javascript
    const sourceRows = (data, onClick) => {
        return data?.grouped && Object.entries(data?.grouped).map(([category, plugs], index) => {
            return <div className="RowGroup" style={{width: "100%"}} key={index}>
                <header>{category}</header>
                <div>
                    {plugs.map((row, subIndex) => {
                        return <BrowserRow key={index + "-" + subIndex}
                                           id={row?.id}
                                           data={row}
                                           onClick={() => onClick(row?.id)}/>
                    })}
                </div>
            </div>
        })
    }
```
Add the icon there. Actually it is in the `BrowserRow`

and call the endpoint `DELETE /resource/{id}`. 

Do not forget to display: 

![Screenshot from 2022-10-04 12-03-14](https://user-images.githubusercontent.com/16271564/193792517-cc936bbc-b08f-46c8-b75f-314a83182714.png)

See `src/components/elements/details/ResourceDetails.js` for how we do it in the resource details page. 

This part should be particularly interested:

```javascript
const onDelete = () => {
        confirm({
            title: "Do you want to delete this resource?",
            description: "This action can not be undone."
        }).then(async () => {
            try {
                const response = await asyncRemote(
                    {
                        url: '/resource/' + data.id,
                        method: "DELETE"
                    })
                if (onDeleteComplete) {
                    onDeleteComplete(response)
                }
            } catch(e) {
                console.error(e)
            }
        }).catch(() => {
        })
    }
```

**Comment by carefreeav09** [comment]
I would like to work on this issue

**Comment by atompie** [comment]
@carefreeav09 I assigned the issue to you

**Comment by atompie** [comment]
Fixed in 0.7.3
